eclipse_plugin/pom.xml

```diff
@@ -48,20 +49,28 @@
     <dependency>
       <groupId>com.google.googlejavaformat</groupId>
       <artifactId>google-java-format</artifactId>
-      <version>1.6</version>
+      <version>${revision}</version>
     </dependency>
   </dependencies>
 
   <build>
     <plugins>
-
       <plugin>
         <groupId>org.eclipse.tycho</groupId>
         <artifactId>tycho-maven-plugin</artifactId>
         <version>${tycho-version}</version>
         <extensions>true</extensions>
       </plugin>
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>11</source>
```

Comment:
`<release>11</release>` would be better, particularly when building with more recent JDK versions as that guarantees that you won't also use JDK12+ APIs.

Reply:
I copied this section from the current core [`pom.xml`](https://github.com/google/google-java-format/blob/master/core/pom.xml#L250).